mixer/pkg/adapter/configError.go

```diff
@@ -75,7 +75,7 @@ func (e *ConfigErrors) Append(field string, err error) *ConfigErrors {
 
 // Error returns a string representation of the configuration error.
 func (e ConfigError) Error() string {
-	return fmt.Sprintf("%s: %s", e.Field, e.Underlying)
+	return fmt.Sprintf("%s: %s", e.Field, e.Underlying.Error())
```

Comment:
This is unnecessary ,  because exist of `func (e ConfigError) String() string {`